terra-cluster/k8s-monitoring.tf

```diff
@@ -3,6 +3,7 @@ module "cluster_monitoring" {
   providers = {
     google.target = google.target
   }
-  project               = var.google_project
+  # Work around for legacy gcp projects being monitored from a common external stackdriver workspace
+  project               = "${var.use_legacy_stackdriver_workspace ? var.stackdriver_workspace_project : var.google_project}"
```

Comment:
Remove quotes?

Reply:
the example of conditionals in the terraform docs had quotes. That might not be up to date though, idk.

Reply:
oh wait nvm, was looking at the wrong example 🤦